js/Source/Slider.Modify/Behavior.Slider.Modify.js

```diff
@@ -31,7 +32,11 @@ Behavior.addGlobalFilter('Slider.Modify', {
     // slideFill is optional
     var slideFill = api.get('fill') ? api.getElement('fill') : null;
     var knob = api.getElement('knob');
+    var offset = api.get('offset');
     var targets = api.getAs(Array, 'targets');
+    var moveClassTargets;
+    if(api.get('moveClassTargets')) moveClassTargets = api.getElements('moveClassTargets');
+    var moveClass = api.get('moveClass');
```

Comment:
I wouldn't declare this variable if you're only using it once. Just put the api call down on line 55...

Up to you. Not a big deal.